packages/core/test/integration/query-interface/add-show-remove-constraint.test.ts

```diff
@@ -7,12 +7,17 @@ const dialect = sequelize.getDialect();
 
 describe('QueryInterface#{add,show,removeConstraint}', () => {
   describe('Without schema', () => {
+    const defaultSchema = dialect === 'db2' ? 'DB2INST1' : sequelize.dialect.getDefaultSchema();
```

Comment:
If this is the actual default schema for DB2 we should add it to `getDefaultSchema` in the index.ts file of DB2

Reply:
According to https://www.ibm.com/docs/en/db2-for-zos/12?topic=concepts-db2-schemas-schema-qualifiers, the default schema is `the authorization ID of the owner of the plan or package`.

Do you know if `the authoriation ID of the owner` is the same as the username that is used in the connection?
In this case the default schema would be equal to the username of the sequelize connection.

Reply:
Unfortunately I don't. But maybe you can play around which some env variables and see which one is changing the schema?

Reply:
Ok so I changed the `DB2INSTANCE` in the `.env_list` file in the `dev/db2` folder to a random value and updated this in the scripts and test config and when logging in to the cli and executing the commands, it appears that the default schema is the username of the user.

Do you want to get this verified by someone that actually uses db2 or just update the `getDefaultSchema` to return the username in the sequelize connection config.

Reply:
Just update it, it seems to correspond with the documentation of db2 as well

Reply:
Done

Reply:
Actually, thinking about this again; can you move this to a separate PR where you also update the other query generator functions that now use `USER` and `CURRENT SCHEMA`? Then that is more aligned with the other dialects

Reply:
Sure, have submitted PR #16307 which implements these changes.